### ISSUE TITLE
Error with FlxRuntimeShader file when compiling

### ISSUE THREAD

**Issue by JMV8712** [FunkinCrew/Funkin#2373]
This is the only error I get when I make attempts to compile, I've followed all the steps to the compiling process, including the visual studio build tools components,

MSVC v143 VS 2022 C++ x64/x86 build tools
Windows 10/11 SDK (windows 10 specifically)

I am not using VS build tools though, I'm doing it straight through command prompt, and the clone repo is on my desktop, I've downloaded all the hmm library contents, and made sure I was in the right directory when I installed.

![Desktop Screenshot 2024 05 08 - 20 37 08 54](https://github.com/FunkinCrew/Funkin/assets/88689478/b06aca6f-c105-4196-9c0f-e12f003d69cb)
 
The screenshot is an error due to the FlxRuntimeShader.hx file, I don't know enough about how haxe works to tell whats going on outside of the fact that the this object on 47 doesn't know what a glVersion is, I doubt tweaking the file is the correct option here.

Anybody got any methods to try to fix?

- [x] Windows
- [ ] Mac
- [ ] Linux
- [ ] HTML5

**Comment by biomseed** [comment]
Make sure you're using the version of flixel-addons from the funkin crew, not the regular one

**Comment by JMV8712** [comment]
Could you give the repo link to the funkin crew's version? If not do I get it through hmm install?

**Comment by METGameDev** [comment]
yeah, its gotten through hmm, but it has a tendency to crash in the middle due to the funkin.vis, so just remove that from the hmm and THEN run it, and you should be good.

**Comment by JMV8712** [comment]
> yeah, its gotten through hmm, but it has a tendency to crash in the middle due to the funkin.vis, so just remove that from the hmm and THEN run it, and you should be good.

When you say remove that, you mean the funkin.vis? Cause I did 'hmm remove funkin.vis', but now it says I need it, I'm hella confused.

**Comment by MulakkJ** [comment]
> yeah, its gotten through hmm, but it has a tendency to crash in the middle due to the funkin.vis, so just remove that from the hmm and THEN run it, and you should be good.

if i remove funkin.vis it says i need it tho

**Comment by moxie-coder** [comment]
> > yeah, its gotten through hmm, but it has a tendency to crash in the middle due to the funkin.vis, so just remove that from the hmm and THEN run it, and you should be good.
> 
> if i remove funkin.vis it says i need it tho

because you’re supposed to reinstall it

**Comment by AbnormalPoof** [comment]
Got this issue once, pretty sure reinstalling `flixel-addons` does the trick (`hmm reinstall flixel-addons -f`).
